modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -45,9 +45,25 @@ For {op-system-base-full} version 8, you can install the {pipelines-title} CLI (
 
 . Enable the repositories required by {pipelines-title}:
 +
+Linux (x86_64, amd64)
```

Comment:
@SNiemann15 as you mentioned, bullets would definitely make it look much better. Do add + on an empty line before the code snippet to ensure they are aligned properly.